.ci/travis/checkstyle_regression_no_exception.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+BRANCH_COMMIT="$1"
+MASTER_COMMIT="$2"
+CHECKSTYLE_VERSION="$3"
+
+if [ ! -d checkstyle ]; then
+    git clone https://github.com/checkstyle/checkstyle
+fi
+
+if [ ! -d contribution ]; then
+    git clone --depth=1 https://github.com/checkstyle/contribution
```

Comment:
> Maybe the cause is testing HEAD and specific commit in one time.

That can be an issue for `checkstyle/checkstyle`. It shouldn't be an issue for contribution which we don't switch branches on.